src/main/java/seedu/toluist/commons/core/Messages.java

```diff
@@ -7,7 +7,7 @@
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
-    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_INVALID_TASK_INDEX = "A valid task index was not provided";
```

Comment:
Do anything you want, can just merge first, i find it pointless to argue over here cuz need type a lot. Shall do it in cs2101 later...